src/helpers/genericArticleHelper.js

```diff
@@ -5,22 +5,22 @@ const { Op } = Sequelize;
 const { Users, Categories, Articles } = models;
```

Comment:
Please rename this file to something like `articleHelper.js`

Reply:
Done.